nemo_skills/inference/model/__init__.py

```
@@ -50,18 +50,20 @@
 }
 
 
-def get_model(server_type, tokenizer=None, **kwargs):
+def get_model(server_type, client_type="chat_completion", tokenizer=None, **kwargs):
     """A helper function to make it easier to set server through cmd."""
     model_class = models[server_type.lower()]
     if server_type == "trtllm" and kwargs.get("enable_soft_fail", False):
         if kwargs.get("context_limit_retry_strategy", None) is not None:
             raise ValueError("context_limit_retry_strategy is not supported for trtllm")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)